kclvm/tools/src/LSP/src/quick_fix.rs

```diff
@@ -125,6 +169,30 @@ pub(crate) fn convert_code_to_kcl_diag_id(code: &NumberOrString) -> Option<Diagn
     }
 }
 
+fn parse_multiple_assignment(error_message: &str) -> Option<(Vec<String>, Vec<String>)> {
```

Comment:
I think this should be done through Parser specific error generation instead of passing code snippets to LSP and then handing them over to LSP for judgment.